test-data/unit/check-attr.test

```diff
@@ -0,0 +1,444 @@
+[case testAttrsSimple]
+# TODO(David):
+# Add tests for:
+# o type inference
```

Comment:
Tests for inference should verify that a generic attrs class has correctly inferred type variables after a call. I think you already have one test, just add one more where it would lead to an error (like conflicting attributes to ``__init__``).